package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "business-edit-ui",
-  "version": "4.1.1",
+  "version": "4.1.2",
```

Comment:
Check with Travis whether you or he should go first. Neither is going to Test today or Prod tomorrow so it's entirely up to you guys.

Reply:
Okay to resolve?

Reply:
Almost. Please update the app version.

Reply:
Done